test/parallel/test-stream-pipeline.js

```diff
@@ -1350,7 +1347,7 @@ const tsp = require('timers/promises');
   });
   const cb = common.mustCall((err) => {
     assert.strictEqual(err.name, 'AbortError');
-    assert.strictEqual(res, '012345');
+    assert.strictEqual(res, '01234');
```

Comment:
why this?

Reply:
There is a timing difference due to how streams are resumed with `'readable'` event.